vm/range_test.go

```diff
@@ -240,37 +240,37 @@ func TestInclude(t *testing.T) {
 	}{
 		{`
 		(5..10).include(10)
-		`, TRUE},
+		`, true},
 		{`
 		(5..10).include(11)
-		`, FALSE},
+		`, false},
 		{`
 		(5..10).include(7)
-		`, TRUE},
+		`, true},
 		{`
 		(5..10).include(5)
-		`, TRUE},
+		`, true},
 		{`
 		(5..10).include(4)
-		`, FALSE},
+		`, false},
```

Comment:
@Maxwell-Alexius Use go's primitive type for testing, we don't need to init goby objects when running tests.